### ISSUE TITLE
BFO_0000070 ("s depends on") obsolete? Replace with RO:0002502?

### ISSUE THREAD

**Issue by chris-grove** [obophenotype/c-elegans-phenotype-ontology#26]
There are several axioms using BFO_0000070 which has no data associated with it. From what I can find, namely this BFO GitHub page:

https://github.com/BFO-ontology/BFO/blob/master/releases/owl-ruttenberg-2010-05-25/bfo-relations.lisp

BFO_0000070 looks like it was at some point "s depends on"

Maybe (although I can't yet find documentation on it) the RO term "depends on" RO:0002502 has replaced it?

**Comment by matentzn** [comment]
Good catch! I do not like this relation in general for EQ. We will go through this tomorrow during the face2face.

**Comment by dosumis** [comment]
I'm not sure about RO:depends_on.

s-depends-on will be in the new BFO.  It is a superProperty of BFO:inheres_in, so that may be a better replacement.

Can you share example of where it's used?

**Comment by chris-grove** [comment]
@dosumis 

1) chemical response WB phenotype terms (many cases, below is one example):

acetylcholinesterase inhibitor hypersensitive (WBPhenotype:0000500)

'has part' some 
('hyperresponsive to' and (BFO_0000070 some 'EC 3.1.1.7 (acetylcholinesterase) inhibitor') and ('inheres in' some 'response to drug'))

2) absent/no <entity> phenotype terms (many cases, below is one example):

alae absent (WBPhenotype:0001228)

'has part' some 
('lacks all parts of type' and (BFO_0000070 some 'cuticular ala'))

3) backing not sustained (WBPhenotype:0001325)

'has part' some 
('having decreased processual parts' and (BFO_0000070 some 'backward locomotion'))

4) connected mitochondria (WBPhenotype:0001402)

'has part' some 
('fused with' and (BFO_0000070 some mitochondrion) and ('inheres in' some mitochondrion))

5) egg laying defective (WBPhenotype:0000006)

'has part' some 
('lacking processual parts' and (BFO_0000070 some oviposition))

6) epidermis cuticle detached (WBPhenotype:0000090)

'has part' some 
('detached from' and (BFO_0000070 some cuticle) and ('inheres in' some hypodermis))

220 uses overall in WBPhenotype

**Comment by matentzn** [comment]
Looking through this, I think the 'BFO70' relation mostly does the job that 'towards' is currently doing for EQ patterns. By default, I would say, replace the relation by 'towards' (RO_0002503), and you would be mostly fine (note I am not saying that using towards is a GOOD choice for the definition, I just think it is what was intended by @cmungall ).  However, occasionally (see below), it is better to use inheres in, in particular when there is no other affected entity in the definition. I think we still need more fundamental discussions of what to do with qualities such as 'lacks all parts of type': Is this a relational quality that indicates that **something** lacks all parts of type something else (then we would need to add an inheres in clause wherever there is a lonely towards/BFO70 clause), of is it something akin to absent -> with the thing that is missing being the primary entity. Does the below make sense @dosumis 

```
acetylcholinesterase inhibitor hypersensitive (WBPhenotype:0000500):
'has part' some
('hyperresponsive to' and (BFO_0000070 some 'EC 3.1.1.7 (acetylcholinesterase) inhibitor') and ('inheres in' some 'response to drug'))
```
--> Here it seems to be that 'towards' (RO_0002503) should have been used.

```
absent/no (entity) phenotype terms (many cases, below is one example):
alae absent (WBPhenotype:0001228)

'has part' some
('lacks all parts of type' and (BFO_0000070 some 'cuticular ala'))
```
--> Should be inheres in? 

```
backing not sustained (WBPhenotype:0001325)
'has part' some
('having decreased processual parts' and (BFO_0000070 some 'backward locomotion'))
```
inheres in.

```
connected mitochondria (WBPhenotype:0001402)
'has part' some
('fused with' and (BFO_0000070 some mitochondrion) and ('inheres in' some mitochondrion))
```
towards

```
egg laying defective (WBPhenotype:0000006)
'has part' some
('lacking processual parts' and (BFO_0000070 some oviposition))
```
inheres in 

```
epidermis cuticle detached (WBPhenotype:0000090)
'has part' some
('detached from' and (BFO_0000070 some cuticle) and ('inheres in' some hypodermis))
```
towards
